sys/include/net/ng_ipv6/hdr.h

```diff
@@ -92,8 +92,8 @@ typedef struct __attribute__((packed)) {
     network_uint16_t len;   /**< payload length of this packet. */
     uint8_t nh;             /**< type of next header in this packet. */
     uint8_t hl;             /**< hop limit for this packet. */
-    ng_ipv6_addr_t src;     /**< source address of this packet. */
-    ng_ipv6_addr_t dst;     /**< destination address of this packet. */
+    ipv6_addr_t src;        /**< source address of this packet. */
+    ipv6_addr_t dst;        /**< destination address of this packet. */
 } ng_ipv6_hdr_t;
```

Comment:
I guess this struct is sensible to non-gnrc implementations as well. Or do you want to separate these things in a separate PR?